pages/hub/apps/guides/azuro-waves.mdx

```diff
@@ -0,0 +1,353 @@
+        <defs>
+          <linearGradient id="sky" gradientUnits="userSpaceOnUse">
+            <stop stopColor="#94D3F3" />
+            <stop stopColor="#83D5FF" offset="100%" />
+          </linearGradient>
+          <linearGradient id="blue" gradientUnits="userSpaceOnUse">
+            <stop stopColor="#83D5FF" />
+            <stop stopColor="#64CAFF" offset="100%" />
+          </linearGradient>
+          <linearGradient id="ultramarine" gradientUnits="userSpaceOnUse">
+            <stop stopColor="#64CAFF" />
+            <stop stopColor="#05AAFF" offset="100%" />
+          </linearGradient>
+          <linearGradient id="bright" gradientUnits="userSpaceOnUse">
+            <stop stopColor="#05AAFF" />
+            <stop stopColor="#3499FE" offset="100%" />
+          </linearGradient>
+          <linearGradient id="brilliant" gradientUnits="userSpaceOnUse">
+            <stop stopColor="#3499FE" />
+            <stop stopColor="#3D67FF" offset="100%" />
+          </linearGradient>
+          <linearGradient id="royal" gradientUnits="userSpaceOnUse">
+            <stop stopColor="#3D67FF" />
+            <stop stopColor="#022E87" offset="100%" />
+          </linearGradient>
+        </defs>
+      </svg>
+    </div>
+  </Tabs.Tab>
+</Tabs>
+
+## Common types
+
+There are common types that are reused in the code examples below.
+
+```ts copy
+export enum WavesLevelName {
+  Grey = 'Grey',
+  Mist = 'Mist',
+  Sky = 'Sky',
+  Blue = 'Blue',
+  Ultramarine = 'Ultramarine',
+  Bright = 'Bright',
+  Brilliant = 'Brilliant',
+  Royal = 'Royal',
+}
+
+export type WavesLevelDescription = {
+  level: number
+  name: WavesLevelName
+  /**  e.g. "1.4" */
+  boost: `${number}`
+  /**  e.g. "50000" */
+  pointsNeeded: `${number}`
+  comment: string
+}
+
+export type WavesParticipant = {
+  address: `0x${string}`
+  waveId: number
+  levelActivated: boolean
+  initialLevel: number
+  level: number
+  levelDescription: WavesLevelDescription
+
+  /** "2.100000", total user points without level multiplier */
+  points: `${number}`
+  /** "2.100000", final points with level multiplier ('boost') */
+  multipliedPoints: `${number}`
+}
+```
+
+## Base API URL
+
+For working with Azuro Waves data, we'll utilize the public API. The base URL is:
```

Comment:
To work with Azuro Waves data, we'll use the public API. ...

Reply:
https://github.com/Azuro-protocol/gem-docs/pull/199/commits/6712abdfb5d5239837b7ffb1b11523d775b8c66d